js/controller.js

```diff
@@ -975,6 +986,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         width: ""
       });
       this.state.forceControlBarVisible = (this.state.pluginsElement.children().length > 0);
+      if (this.state.mainVideoPlayhead > 0) {
+        this.isNewVrVideo = false;
```

Comment:
What is isNewVrVideo used for? Also, why is this check in onWillPlayAds?

Reply:
We use it for checking if needed to show some notifications to user about type of video. 
We change the value in onWillPlayAds for prevent chaw extra notifications to user.